.github/workflows/macos-ci.yml

```diff
@@ -45,8 +45,13 @@ jobs:
       # - libpng
       # - libvorbis
       # - python
+      - name: Setup Python
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+        with:
+          python-version: 3.12.8
+      
       - name: Install dependencies using homebrew
-        run: brew install boost-python3 gtk+3 gtkglext sdl
+        run: brew install boost-python3 gtk+3 gtkglext sdl libvorbis
```

Comment:
Apparently, libvorbis is no longer preinstalled on the latest macOS runners. We have to install it ourselves now.